.vib/grafana/cypress/cypress/integration/grafana_test_suite.js

```diff
@@ -0,0 +1,146 @@
+/// <reference types="cypress" />
+import {
+  deleteDashboard,
+  deleteAPIKey,
+  clickDeleteButton,
+  verifySuccesOfAction
+} from './utils'
+
+it('allows user to log out', () => {
+  cy.login();
+  cy.visit('/logout');
+  cy.get('.login-content-box').should('exist');
```

Comment:
I think `cy.get(...)` already checks whether an element in the DOM exists or not. Hence `should('exist')` should be kind of redundant here. 

Instead (if it applies) you can go with `should('be.visible');`